pack/tdc/tdcp.json

```diff
@@ -0,0 +1,553 @@
+[
+    {
+        "back_flavor": "",
```

Comment:
```suggestion
        "back_flavor": "<b>This is placeholder flavor text until the card is revealed.</b>",
```

Reply:
Resolved as per discord: we extract george to its own PR